build.cake

```diff
@@ -0,0 +1,161 @@
+var deployDir = Directory("./deploy");
+var zipFileHash = "";
+
+var squirrelDeployDir = deployDir + Directory("Squirrel");
+var squirrelReleaseDir = squirrelDeployDir + Directory("Releases");
+
+Task("Clean")
+    .Does(() =>
+    {
+		Func<IFileSystemInfo, bool> excludeSquirrelDir =
+			fileSystemInfo => !(fileSystemInfo.Path.FullPath.IndexOf("Squirrel", StringComparison.OrdinalIgnoreCase) >= 0);
+        
+		CleanDirectory(buildDir);
+		CleanDirectory(deployDir, excludeSquirrelDir);
+    });
+
+Task("Restore-NuGet-Packages")
+    .IsDependentOn("Clean")
+    .Does(() =>
+    {
+        NuGetRestore(solutionFile);
+    });
+
+Task("Build")
+    .IsDependentOn("Restore-NuGet-Packages")
+    .Does(() => 
+    {
+        MSBuild(solutionFile, settings =>
+            settings.SetConfiguration(configuration));
+    });
+
+Task("Run-Unit-Tests")
+    .IsDependentOn("Build")
+    .Does(() =>
+    {
+        XUnit("./src/Carnac.Tests/bin/" + configuration + "/*.Tests.dll");
+    });
+
+Task("Package-Squirrel")
+	.IsDependentOn("Run-Unit-Tests")
+	.Does(() =>
+	{
+		var syncReleasesDir = toolsDir + Directory("squirrel.windows/tools");
+
+		EnsureDirectoryExists(deployDir);
+		EnsureDirectoryExists(squirrelDeployDir);
+
+		// Create nuget package
+		var releaseFiles = new HashSet<string>(
+				GetFiles(buildDir.Path + "/**/*.*").Select(f => f.FullPath)
+					.Concat(GetFiles((toolsDir + Directory("DeltaCompressionDotNet/lib/net45")).Path + "/*.dll").Select(f => f.FullPath)
+						.Concat(GetFiles((toolsDir + Directory("Mono.Cecil/lib/net45")).Path + "/*.dll").Select(f => f.FullPath)
+							.Concat(GetFiles((toolsDir + Directory("Splat/lib/Net45")).Path + "/*.dll").Select(f => f.FullPath)
+								.Concat(GetFiles((toolsDir + Directory("squirrel.windows/lib/Net45")).Path + "/ICSharpCode.SharpZipLib.*").Select(f => f.FullPath)
+									.Concat(GetFiles((toolsDir + Directory("squirrel.windows/lib/Net45")).Path + "/*Squirrel.dll").Select(f => f.FullPath))
+								)
+							)
+						)
+					)
+			);
+		releaseFiles.RemoveWhere(f => f.Contains(".vshost.") || f.EndsWith(".pdb"));
+
+		var nuGetPackSettings = new NuGetPackSettings
+		{
+			Version = version,
+			Files = releaseFiles.Select(f => new NuSpecContent { Source = f, Target = "lib/net45" + (f.Contains("Keymaps") ? "/Keymaps" : "") }).ToList(),
+			BasePath = buildDir,
+			OutputDirectory = squirrelDeployDir,
+			NoPackageAnalysis = true
+		};
+		NuGetPack("./src/Carnac/Carnac.nuspec", nuGetPackSettings);
+		
+		// Sync latest release to build new package
+		var squirrelSyncReleasesExe = syncReleasesDir + File("SyncReleases.exe");
+		StartProcess(squirrelSyncReleasesExe, new ProcessSettings { Arguments = "--url " + githubRepoUrl + " --releaseDir " + squirrelReleaseDir.Path + (!string.IsNullOrEmpty(githubAuthToken) ? " --token " + githubAuthToken : "") });
+
+		// Create new squirrel package
+		Squirrel(
+			squirrelDeployDir + File("carnac." + version + ".nupkg"), 
```

Comment:
Why, yes we can...just need to use the `-Experimental` flag on `Cake` apparently 😄